core/app/models/spree/stock_location.rb

```diff
@@ -106,8 +106,18 @@ def fill_status(variant, quantity)
     private
 
     def create_stock_items
-      Variant.includes(:product).find_each do |variant|
-        propagate_variant(variant)
+      prepared_stock_items = Spree::Variant.ids.map do |variant_id|
+        Hash[
+          'stock_location_id', id,
+          'variant_id', variant_id,
+          'backorderable', backorderable_default,
+          'created_at', DateTime.current,
+          'updated_at', DateTime.current
+        ]
+      end
+      if prepared_stock_items.any?
+        stock_items.insert_all(prepared_stock_items)
+        Spree::Variant.touch_all
```

Comment:
```suggestion
        variants_scope.touch_all
```

Reply:
This doesn't seem to work. I don't think we have such scope in StockLocation.

Reply:
@aplegatt sorry suggestion didn't add earlier - let's add it here: https://github.com/spree/spree/pull/10333#pullrequestreview-445748671